R/loop_tasks.R

```diff
@@ -83,7 +83,8 @@ loop_tasks <- function(
   total_num_tasks_complete <- 0
   if (verbose){
     pb <- progress::progress_bar$new(
-      format = ":what [:bar] :percent",
+      format = ":what [:bar] :percent  ",
+      width = getOption('width')+4,
```

Comment:
progress bars don't generally play well with other `message()`s, but this formatting makes it more likely that if you do have other `message` calls within your task steps, they'll at least be readable if not beautifully placed.

Reply:
Yes, I realized that there may actually be cases when we want verbose messages but not the progress bar after all. This is a good fix, but we might want to revisit having 3 (or 4?) message output options, including silent.

Reply:
Agreed...I thought about pushing for that as I was struggling through this today, but this formatting change actually gets us pretty close, and I liked your suggestion to keep this simple if we can. That said, adding verbosity levels wouldn't be too tough, either, and could clean it up even more. I suggest that we keep this possibility in mind and then wait on it until it feels urgent (and then if it does, do it!).